.github/workflows/multi-platform-docker-build.yml

```diff
@@ -0,0 +1,110 @@
+name: crtsh-exporter-multi-platform-docker-build
+
+on:
+  push:
+    # tags:
+    #   - "v*.*.*" 
+    branches:
+      - master
+
+  schedule:
+    # This will run at 11:00 PM UTC every Monday.
+    - cron: '0 23 * * 1'
+jobs:
+  crtsh-exporter-multi-platform-docker-build:
+    name: crtsh-exporter
+    runs-on: ubuntu-latest
+    env:
+      REGISTRY: "ghcr.io" 
+      REPO: "${{ github.repository_owner }}/crtsh-exporter" 
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Set SHA_SHORT env var and get Kernel version
+        id: vars
+        run: |
+          echo "SHA_SHORT=$(git rev-parse --short HEAD)" >> ${GITHUB_ENV}
+          echo "VERSION=$(uname --kernel-release)" >> ${GITHUB_ENV}
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+      
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+
+      - name: Login
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GHCR }}
+          
+      - name: Buildx linux/amd64 Docker Image
+        uses: docker/build-push-action@v3
+        with:
+          context: .
+          platforms: linux/amd64
+          build-args: |
+            VERSION=${{ env.SHA_SHORT }}
+            COMMIT=${{ env.REPO }}
```

Comment:
Why `{{ env.REPO }}` here but `${{ env.SHA_SHORT }}` and `${{ github.SHA }}` for the other platforms?

Reply:
This was on oversight, new commit updates it to be `COMMIT=${{ github.sha }}`